manimlib/camera/camera.py

```diff
@@ -26,6 +26,19 @@
 
 
 class Camera(object):
+    """
+    Base Camera class.
+    This is the object which takes care of what exactly is displayed
+    on screen at any given moment.
+
+    Some important CONFIG values and local variables to note are:
+
+    self.background_image : str, optional
```

Comment:
Wrong format for attributes. (Note: with dataclasses we will be able to deal with this more appropriately.)
For now...

```py
"""(...)
    Attributes
    ----------
        background_image : str, optional
            The path to an image that should be the background image.
            If not set, the background is filled with `self.background_color`
    
        pixel_height : int
            (Text here???)
"""
```